--- CMakeLists.txt	2021-09-09 01:48:37.900560551 -0700
+++ myCMakeLists.txt	2021-09-09 13:46:05.445403333 -0700
@@ -13,9 +13,13 @@
 
 # enforce matching of the CXX and Fortran compilers
 if (NOT "${CMAKE_Fortran_COMPILER_ID}" MATCHES "${CMAKE_CXX_COMPILER_ID}")
-    message(FATAL_ERROR "You need to use the same vendor for your C++ and Fortran compiler")
+    message(STATUS "You should to use the same vendor for your C++ and Fortran compiler")
 endif ()
 
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${PROJECT_SOURCE_DIR}/lib)
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${PROJECT_SOURCE_DIR}/lib)
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_SOURCE_DIR}/bin)
+
 # flags for all three compiler types
 # todo: inspect the flags and implement in a nicer way
 if ("${CMAKE_Fortran_COMPILER_ID}" MATCHES "Intel")
